wazimap_np/templates/profile/sections/education.html

```diff
@@ -0,0 +1,54 @@
+<article id="education" class="clearfix">
+    {% if education.area_has_data == True %}
+        <header class="section-contents">
+            <h1>Education</h1>
+        </header>
+        <div class="section-container">
+
+            <section class="clearfix stat-row">
+                <h2><a class="permalink" href="#literacy-by-sex"
+                       id="literacy-by-sex">Literacy <i class="fa fa-link"></i></a>
+                </h2>
+                <div class="column-third"
+                     id="chart-pie-education-literacy_distribution"
+                     data-stat-type="percentage"></div>
+                <div class="column-two-thirds"
+                     id="chart-grouped_column-education-literacy_by_sex_distribution"
+                     data-stat-type="scaled-percentage"
+                     data-chart-title="Literacy rate by sex"
+                     data-qualifier="Universe: {{ education.literacy_by_sex_distribution.metadata.universe }}"></div>
+            </section>
+
+            <section class="clearfix stat-row">
+                <h2><a class="permalink" href="#education-level-reached"
+                       id="education-level-reached">Education Level Passed <i
+                        class="fa fa-link"></i></a></h2>
+                <div class="column-third">
+                    {% include 'profile/_blocks/_stat_list.html' with stat=education.primary_level_reached stat_type='percentage' %}
+                </div>
+                <div class="column-two-thirds"
+                     id="chart-histogram-education-education_level_passed_distribution"
+                     data-stat-type="scaled-percentage"
+                     data-chart-title="Highest education level reached"
+                     data-qualifier="Universe: {{ education.education_level_passed_distribution.metadata.universe }}"></div>
+            </section>
+            <section class="clearfix stat-row">
+                <h2><a class="permalink" href="#education-level-reached-by-sex"
+                       id="education-level-reached-by-sex">Education Level
+                    Passed by Sex <i class="fa fa-link"></i></a></h2>
+                <div class="column-third">
+                    {% include 'profile/_blocks/_stat_list.html' with stat=education.primary_level_reached_by_sex stat_type='percentage' %}
+                </div>
+                <div class="column-two-thirds"
+                     id="chart-grouped_column-education-education_level_by_sex_distribution"
+                     data-stat-type="scaled-percentage"
+                     data-chart-title="Highest education level reached"
+                     data-qualifier="Universe: {{ education.education_level_by_sex_distribution.metadata.universe }}"></div>
+                <small>Source: <br>
+                    <a href="http://nationaldata.gov.np/" target="_blank">National Data Profile</a>
```

Comment:
When the link is here, I think that it gives the impression that only the "Education Level Passed by Sex" is from the National Data Profile. Since there is not a summary section for education, perhaps it would make sense to include the link with every section.

Reply:
Done.